uiowa/uiowa
# Website reporting tool

Provides a BLT command that generates a report of website analytics data.

Adds drush aliases for D7 applications.

# How to test
```
blt ura --export
```
- Observe that the file gets created.
- And gets updated with new records as domains are checked.
- No need to run the entire thing.
